### PR TITLE
Adding Comments plugin

### PR DESCRIPTION
**Pull request by Kareylo**
Just added a plugin i just finish to comment whatever model we want !

https://github.com/Kareylo/CakePHP-Comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +230,7 @@ Additional lists you might find useful:
 - [Ajax plugin](https://github.com/dereuromark/cakephp-ajax) - A plugin to ease handling AJAX requests.
 - [CakeAdmin plugin](https://github.com/cakemanager/cakephp-cakeadmin) - A non-stable user management plugin with a built-in admin area.
 - [CakeMiddlewares](https://github.com/chrisShick/CakeMiddlewares) - A collection of Cakephp Middlewares. Requires CakePHP version 3.4+.
+- [Comments plugin](https://github.com/Kareylo/CakePHP-Comments) - A fully customizable Comments plugin. Requires CakePHP version 3.3.2+.
```
Comment (dereuromark) [inline README.md]: The versions can be removed IMO. thats part of the readme and composer constraint.

**Comment by dereuromark** [comment]
A few tests here would be nice to avoid regression.

**Comment by Kareylo** [comment]
I begin to work on the differents tests. Will push them in the next release of the plugin when it's done.

**Comment by Kareylo** [comment]
After a hard fight against travis, tests passed !

**Comment by ADmad** [comment]
- If  you remove the `id` field values from your fixtures your test suite will then run properly on postgres too. Postgres doesn't like autoincrement values being set before hand. 
- Readme is missing instructions for loading the plugin (it might seem obvious but newbies can trip on it)
- I would suggest removing instructions of setting `$this->helpers` in controller. It's preferred to load behavior through `AppView` instead.

**Comment by Kareylo** [comment]
@ADmad 
* If I remove the `id` field, every test using the field fail. Any example on how to setup that ?
* Forget to add that, my bad.
* Totally agree with you, directly remove from the readme

**Comment by ADmad** [comment]
> If I remove the id field, every test using the field fail. Any example on how to setup that ?

Check the core fixtures or any plugin of FriendsOfCake.

**Comment by dereuromark** [comment]
For me removing the versions would make this mergeable.

**Pull request merged**